vital/algo/triangulate_landmarks.h

```diff
@@ -43,34 +47,38 @@ class VITAL_ALGO_EXPORT triangulate_landmarks
    * landmark map and which have support in the feature tracks and cameras
    */
   virtual void
-  triangulate(kwiver::vital::camera_map_sptr cameras,
-              kwiver::vital::feature_track_set_sptr tracks,
-              kwiver::vital::landmark_map_sptr& landmarks) const = 0;
+  triangulate( kwiver::vital::camera_map_sptr cameras,
+               kwiver::vital::feature_track_set_sptr tracks,
+               kwiver::vital::landmark_map_sptr& landmarks ) const = 0;
+
+  /// Triangulate the landmark locations given sets of cameras and feature
+  /// tracks
 
-  /// Triangulate the landmark locations given sets of cameras and feature tracks
   /**
-  * \param [in] cameras the cameras viewing the landmarks
-  * \param [in] tracks the feature tracks to use as constraints stored in a track map
-  * \param [in,out] landmarks the landmarks to triangulate
-  *
-  * This function only triangulates the landmarks with indices in the
-  * landmark map and which have support in the feature tracks and cameras.
-  */
+   * \param [in] cameras the cameras viewing the landmarks
+   * \param [in] tracks the feature tracks to use as constraints stored in a
+   * track map
+   * \param [in,out] landmarks the landmarks to triangulate
+   *
+   * This function only triangulates the landmarks with indices in the
+   * landmark map and which have support in the feature tracks and cameras.
+   */
   virtual void
-  triangulate(vital::camera_map_sptr cameras,
-              vital::track_map_t tracks,
-              vital::landmark_map_sptr& landmarks) const;
+  triangulate( vital::camera_map_sptr cameras,
+               vital::track_map_t tracks,
+               vital::landmark_map_sptr& landmarks ) const;
 
 protected:
   triangulate_landmarks();
-
 };
 
 /// type definition for shared pointer to a triangulate landmarks algorithm
-typedef std::shared_ptr<triangulate_landmarks> triangulate_landmarks_sptr;
+typedef std::shared_ptr< triangulate_landmarks > triangulate_landmarks_sptr;
 
 } // end namespace algo
```

Comment:
comment can just be // namespace for consistency
```suggestion
} // namespace algo
```